File: 2-semestre/uri-online-judge/javascript/1113.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

do {
    let linha = lines.shift().split(" ");
    x = parseInt(linha[0]);
    y = parseInt(linha[1]);

    if (x == y) break;
    else if (x < y) console.log("Crescente");
    else console.log("Decrescente");
} while (x != y);

File: 2-semestre/uri-online-judge/c/1113.c
#include <stdio.h>
 
int main() {
    int x, y;

    do {
        scanf("%d %d", &x, &y);

        if (x == y) break;
        else if (x < y) printf("Crescente\n");
        else printf("Decrescente\n");
    } while (x != y);

    return 0;
}
